src/modules/dashboard/components/PermissionManagementDialog/PermissionManagementDialog.tsx

```diff
@@ -242,6 +306,11 @@ const PermissionManagementDialog = ({
                   text={{ id: 'button.confirm' }}
                   type="submit"
                   style={{ width: styles.wideButton }}
+                  disabled={
+                    !userHasPermission ||
+                    !isValid ||
+                    isEqual(sortBy(values.roles), sortBy(initialValues.roles))
+                  }
```

Comment:
This does not do proper validation yet.

Here's two cases I found:

1. I can remove **my own** permissions in "Root" making the colony basically unusuable.

![Screenshot from 2021-02-10 17-19-17](https://user-images.githubusercontent.com/1193222/107534339-7a096780-6bc8-11eb-933e-8815ac3cadd2.png)
![Screenshot from 2021-02-10 17-19-54](https://user-images.githubusercontent.com/1193222/107534341-7b3a9480-6bc8-11eb-853e-33f082adfbf1.png)
![Screenshot from 2021-02-10 17-19-57](https://user-images.githubusercontent.com/1193222/107534343-7bd32b00-6bc8-11eb-9ef5-75d14e3d562b.png)

Ask product about this, but I don't think this is what they want.

2. It seems I cannot remove some user permissions: If the user has a single permission, de-selecting the check box, does not light up the "Confirm" button:

![Screenshot from 2021-02-10 17-35-22](https://user-images.githubusercontent.com/1193222/107534581-aae99c80-6bc8-11eb-883c-30834607a469.png)
![Screenshot from 2021-02-10 17-35-26](https://user-images.githubusercontent.com/1193222/107534583-ab823300-6bc8-11eb-9842-0f3fe432b773.png)

Please be more thorough when doing form validations

Reply:
Consulted with product about the first case. They said we keep it that way, if someone wants to remove their permissions then we shouldn't prevent that

Reply:
Ok then.

In that case just take care of the second case